internal/profile/_static/kibana_config_80.yml

```diff
@@ -1,15 +1,20 @@
 server.name: kibana
 server.host: "0.0.0.0"
+server.ssl.enabled: true
+server.ssl.certificate: "/usr/share/kibana/config/certs/cert.pem"
+server.ssl.key: "/usr/share/kibana/config/certs/key.pem"
+server.ssl.certificateAuthorities: ["/usr/share/kibana/config/certs/ca-cert.pem"]
 
-elasticsearch.hosts: [ "http://elasticsearch:9200" ]
+elasticsearch.hosts: [ "https://elasticsearch:9200" ]
+elasticsearch.ssl.certificateAuthorities: "/usr/share/kibana/config/certs/ca-cert.pem"
 elasticsearch.serviceAccountToken: "AAEAAWVsYXN0aWMva2liYW5hL2VsYXN0aWMtcGFja2FnZS1raWJhbmEtdG9rZW46b2x4b051SWNRa0tYMHdXazdLWmFBdw"
 
 monitoring.ui.container.elasticsearch.enabled: true
 
-xpack.fleet.registryUrl: "http://package-registry:8080"
+xpack.fleet.registryUrl: "https://package-registry"
```

Comment:
In the docker compose network it is listening on `443`, but in the host it is in `8443`. The connection configured here goes through the docker network.

Actually this is something I left halfway and forgot. I considered to use standard ports (443) in all services, only exposing them in the current ports for connections from the host. At the end I only did it for the package-registry because it seems that for other services the same port is used in documentation with and without TLS.

I don't have a strong opinion about this, I can move back to `8080`, use `8443` for the internal network too, or move everything to `443` for the internal network.

Reply:
I don't know what's the Elastic standard, whether we should use the same port (e.g. 9200) for HTTP and HTTPS, or should we switch to a different one, 92xx.

Not a major issue, you can leave it as is. I used to configure consistent ports, so 8443 externally would be 8443 internally.

Reply:
Yeah, I will change this back :+1:

Reply:
Changed back to 8080, another point in favour of using the same ports as before is that we avoid conflicts with other ports users may be used to use.